packages/client/wallets/smart-wallet/src/index.ts

```diff
@@ -14,14 +14,7 @@ export type {
 export type { TransferType, ERC20TransferType, NFTTransferType, SFTTransferType } from "./types/Tokens";
 
 export {
-    TransferError,
-    CrossmintServiceError,
     SmartWalletError,
-    JWTDecryptionError,
-    JWTExpiredError,
-    JWTIdentifierError,
-    JWTInvalidError,
-    NotAuthorizedError,
```

Comment:
Probably still want to export these from this package, still use for a dev and we don't want people to have to download both packages ideally.